testutils/collection/data.go

```diff
@@ -58,19 +58,19 @@ var (
 	SampleFileShareProfiles = []model.ProfileSpec{
 		{
 			BaseModel: &model.BaseModel{
-				Id: "1106b972-66ef-11e7-b172-db03f3689c9c",
+				Id: "2106b972-66ef-11e7-b172-db03f3689c9c",
```

Comment:
Same as above...Why should there be these chages for ID? Let's discuss it. Generator and persisted across tests

Reply:
Same as above file profile has been created earlier it was using block profile